src/java/org/apache/cassandra/config/Config.java

```diff
@@ -543,9 +543,12 @@ public static class SSTableConfig
     public volatile DurationSpec.IntMinutesBound index_summary_resize_interval = new DurationSpec.IntMinutesBound("60m");
 
     @Replaces(oldName = "gc_log_threshold_in_ms", converter = Converters.MILLIS_DURATION_INT, deprecated = true)
-    public volatile DurationSpec.IntMillisecondsBound gc_log_threshold = new DurationSpec.IntMillisecondsBound("200ms");
+    public volatile DurationSpec.IntMillisecondsBound gc_log_threshold = new DurationSpec.IntMillisecondsBound("20s");
```

Comment:
@jmckenzie-dev curious for the reasoning behind raising this default threshold by so much? same question for `gc_warn_threshold`? is it specific to ZGC use? From the comments below it seems like thats the case but wanted a bit more context

Reply:
A big difference regarding GC duration comparing stop-the-world GC to ZGC is that STW GC operations try to finish GC pauses as fast as possible with as many threads as possible, to reduce latency impact due to pausing the program. If it takes too long, advanced STW collectors like G1 will try to be incremental instead, splitting the work over multiple GC pauses. ZGC, being a concurrent collector, aims at doing the opposite: use as few threads as we can get away with, preferring to take longer instead.  There is also no need for incremental GC in the same way. Here the roles are quite reversed: we don't pause the program and hence don't improve latency by using more threads. Conversely, in the concurrent context, latencies get worse when we throw more threads at the problem, due to stealing CPU resources from the program threads. ZGC prefers to stay in the background chipping away at the GC work with a single thread constantly running, compared to having bursts of CPU utilization in order to finish doing GC faster than was really needed. Therefore, GC duration will look longer, but is also not really going to be indicative of any problem the way that it might have been before.